sources/utils.go

```diff
@@ -43,3 +46,8 @@ func HealthyHealthCheckResult(checkType health.CheckType) health.HealthCheckResu
 		State: health.New_HealthState(health.HealthState_HEALTHY),
 	}
 }
+
+func ErrorToUnderlyingSafeParamsMap(err error) map[string]interface{} {
```

Comment:
nit: `SafeParamsFromError` to match the naming style of the library?

Reply:
Done